cmd/lncli/main.go

```diff
@@ -34,6 +34,18 @@ const (
 	defaultMacaroonFilename = "admin.macaroon"
 	defaultRPCPort          = "10009"
 	defaultRPCHostPort      = "localhost:" + defaultRPCPort
+
+	envVarRPCServer       = "LNCLI_RPCSERVER"
+	envVarLNDDir          = "LNCLI_LNDDIR"
+	envVarSOCKSProxy      = "LNCLI_SOCKSPROXY"
+	envVarTLSCertPath     = "LNCLI_TLSCERTPATH"
+	envVarChain           = "LNCLI_CHAIN"
+	envVarNetwork         = "LNCLI_NETWORK"
+	envVarMacaroonPath    = "LNCLI_MACAROONPATH"
+	envVarMacaroonTimeout = "LNCLI_MACAROONTIMEOUT"
+	envVarMacaroonIP      = "LNCLI_MACAROONIP"
+	envVarProfile         = "LNCLI_PROFILE"
+	envVarMacFromJAR      = "LNCLI_MACFROMJAR"
```

Comment:
micro nit: In this context, jar isn't an acronym (not Java ARchive) but the glass container you might put baked cookies in to keep them fresh. So can just be `envVarMacFromJar` ;-)

Reply:
oh I will fix that in a minute, I am not 100% familiarised with 100% of the flags. That will be fixed and squashed again in a min.